p/scripts/main.js

```diff
@@ -1286,7 +1277,57 @@ function init_crypto_form() {
 }
 //</crypto form (Web login)>
 
+var $sidebar = null;
+var useJsScrollbar = true;
+try {
+	useJsScrollbar = !(parseInt(getComputedStyle(document.getElementById('sidebar'), '::-webkit-scrollbar').width) < 99);
```

Comment:
A few notes:
* This code will be safely executed after CSS loading thanks to the script `defer` attribute, and may be executed before jQuery is ready as there is no use of jQuery there.
* `parseInt()` will return `NaN` when the property is empty (e.g. in Firefox), and `NaN < 99 == false`;
* Edge / IE would return the width of the whole sidebar, which is about ~300.

Reply:
Forgive me if there was a way to merge in this change; I added it manually since I didn't see how to.

Reply:
I believe a simple `git pull` should have worked :-)

Reply:
That was what I thought, and what I'd tried, but it didn't seem to pull it! I was probably just doing something wrong, still getting used to git.